protocols/v2/codec-sv2/src/decoder.rs

```diff
@@ -1,3 +1,43 @@
+//! # Stratum V2 Codec Decoder
```

Comment:
"Stratum V2 Codec Decoder" feels a bit redundant.

```suggestion
//! # Stratum V2 Codec Decoder
//! # Decoder
```